comms/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o /comms-linux
 # COPY db/migrations ./db/migrations
 
 EXPOSE 8925
+EXPOSE 8926
```

Comment:
storage is currently using `8926` 😢  will have to roll forward to 8927!

Reply:
https://github.com/AudiusProject/audius-protocol/pull/4823/commits/6e13ce13fdbdebfae956c6d65cf1ff900d372c00